ros_bt_py/ros_bt_py/tree_manager.py

```diff
@@ -371,8 +371,6 @@ def __init__(
         if tick_frequency_hz == 0.0:
             tick_frequency_hz = 10.0
 
-        self.tick_sliding_window = [tick_frequency_hz] * 10
```

Comment:
I like removing the sliding window stuff, that is something for the monitoring GUI to do if we even want it